tests/resources/functional/snowflake/test_lateral_struct/test_lateral_struct_10.sql

```diff
@@ -12,3 +12,8 @@
 SELECT tt.id, FROM_JSON(tt.details, {TT.DETAILS_SCHEMA}) FROM prod.public.table AS tt 
         LATERAL VIEW EXPLODE(FROM_JSON(FROM_JSON(tt.resp, {TT.RESP_SCHEMA}).items, {JSON_COLUMN_SCHEMA})) AS lit
         LATERAL VIEW EXPLODE(FROM_JSON(lit.value.details, {JSON_COLUMN_SCHEMA})) AS ltd;
+
```

Comment:
create a separate file for the variant type. these functional tests would essentially be split by `-- databricks sql:` comment for simplicity sake. apply to other files as well.

Reply:
I believe this is a better design so my corpous of SQLs irrespective of source should generate same query, But I agree the parser needs to be better